libs/market-list/src/lib/components/landing/select-market-list.tsx

```diff
@@ -0,0 +1,117 @@
+// Sparkline for the Market
+
+import { Arrow, Sparkline } from '@vegaprotocol/ui-toolkit';
+import { useRouter } from 'next/router';
+import { mapDataToMarketList } from '../../utils';
+import type { MarketList } from '../__generated__/MarketList';
+
+export interface MarketSparklineProps {
+  candles?: {
+    close: string | undefined | number;
+    open: string | undefined | number;
+  }[];
+}
+
+export const MarketSparkline = ({ candles }: MarketSparklineProps) => {
+  return (
+    <Sparkline
+      width={100}
+      height={20}
+      data={
+        candles
+          ?.filter((m) => !isNaN(Number(m.close)))
+          ?.map((m) => Number(m.close)) || []
+      }
+    />
+  );
+};
+
+// Select Market List
+
+export interface SelectMarketListProps {
+  data: MarketList | undefined;
+}
+
+export const SelectMarketList = ({ data }: SelectMarketListProps) => {
+  const thClassNames = (direction: 'left' | 'right') =>
+    `px-8 text-${direction} font-sans font-normal text-ui-small leading-9 capitalize mb-0 text-dark/80 dark:text-white/80`;
+  const tdClassNames =
+    'px-8 font-sans leading-9 capitalize text-ui-small text-right';
+  const priceChangeClassNames = (value: number) =>
+    value === 0
+      ? 'text-black dark:text-white'
+      : value > 0
+      ? `text-green`
+      : `text-red`;
+  const boldUnderlineClassNames =
+    'px-8 underline font-sans text-base leading-9 font-bold tracking-tight decoration-solid text-ui light:hover:text-black/80 dark:hover:text-white/80';
+
+  const { pathname, push } = useRouter();
+
+  return (
+    <div className="max-h-[40rem] overflow-x-auto">
+      <table className="relative h-full min-w-full whitespace-nowrap">
```

Comment:
I wanted to make this reusable but only the top table styling is really good to re-use. the content is quite custom. not sure if it is worth it.